projects/go-lib/src/lib/components/go-copy/go-copy.component.html

```diff
@@ -0,0 +1,4 @@
+<div (click)="copyStringToClipboard($event.currentTarget)">
```

Comment:
We need to remove the id from the textarea here so we avoid having duplicate ids on the page. We could also clean up our BEM selectors if we structure things like the following:
```html
<div class="go-copy">
  <go-icon icon="file_copy"></go-icon>
  <textarea class="go-copy__textarea">{{text}}</textarea>
</div>
```